src/components/Header/Header.tsx

```diff
@@ -9,7 +9,7 @@ import { useLocalization } from '../../context/localization-context';
 
 function Header() {
   const [isPageScrolled, setIsPageScrolled] = useState(false);
-  const { i18n, language } = useLocalization();
+  const { dataLang, language } = useLocalization();
```

Comment:
Every time you use _**dataLang**_ and _**language**_ together.  It will be better if you hide _**dataLang[language]**_ in LocalizationContext and only get the result.